### DISCUSSION TITLE
А как отключить запрет если в данный момент он не нужен.

### DISCUSSION THREAD

**Discussion by KillerQuQ** [Flowseal/zapret-discord-youtube#826] [Q&A]
А как отключить запрет если в данный момент он не нужен. Лично мне он нужен буквально на час в день, подскажите как его отключить?

**Comment by Eptiy** [comment]
А зачем его отключать? Если у тебя, допустим, стоит discord.bat как сервис, то он больше ни на что, кроме дискорда, не влияет и ни с чем не конфликтует.

**Comment by Flowseal** [comment]
service_remove.bat
